### PR TITLE
Update build matrix

### PR DESCRIPTION
**Pull request by jamesmyatt**
This is all of the stable versions of Python for Linux currently available on Travis CI

Also tidy install_requires

### CODE REVIEW

**Comment by jamesmyatt** [comment]
Note that "python setup.py test" is now bad practice, and doesn't actually do anything

**Review by Danielsoeller — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
 python:
-  - "2.6"
   - "2.7"
-  - "3.2"
-  - "3.3"
+  - "3.4"
+  - "3.5"
+  - "3.6"
+  - "3.7"
+  - "3.8"
```
Comment (Danielsoeller) [inline .travis.yml:8]: - "3.9"

Comment (jamesmyatt) [reply .travis.yml:8]: Done 👍

File: .travis.yml [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
+install:
+  - "python -m pip install -U pip"
+  - "python -m pip install -e ."
 
-# command to run tests
-script: "python setup.py test"
+# commands to run tests
+script:
+  - python -c "import Levenshtein"
+  - "python setup.py test"
```
Comment (jamesmyatt) [inline .travis.yml:21]: BTW, `setup.py test` is deprecated.

**Pull request closed without merging**